acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -6,6 +6,9 @@
 
 # This file contains general test helper methods for pxp-agent acceptance tests
 
+# The standard path for git checkouts is where pcp-broker will be
+GIT_CLONE_FOLDER = '/opt/puppet-git-repos'
```

Comment:
The comment says it's standard, so doesn't the place where it's standardised export a constant?